### DISCUSSION TITLE
Lacking proper documentation for alerting functions

### DISCUSSION THREAD

**Discussion by jzhao-roblox** [grafana/grafana#40306] [Alerting]
I noticed that in the alert creation documentation (https://grafana.com/docs/grafana/latest/alerting/old-alerting/create-alerts/), there is no explanation for individual alert functions. While avg() or max() might be more obvious, I think documentation for functions like diff() would definitely be helpful.

**Comment by osg-grafana** [comment]
cc @achatterjee-grafana

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
